pkg/alerts/nrql_conditions.go

```diff
@@ -357,7 +357,7 @@ func (a *Alerts) CreateNrqlConditionBaselineMutation(
 // UpdateNrqlConditionBaselineMutation updates a baseline NRQL alert condition via New Relic's NerdGraph API.
 func (a *Alerts) UpdateNrqlConditionBaselineMutation(
 	accountID int,
-	conditionID string, // GraphQL scalar type `ID` is a string in JSON
+	conditionID int,
```

Comment:
I don't think we should change this type.  Today IDs look like ints, but that could bite us down the road (which is why the scalar type ID in GraphQL is an opaque string)

Reply:
Just closing the loop for reference later if needed. Spoke about this a bit more and we can keep it like this for since this is more consistent with what we're doing now. We shall see what "the future" holds 🔮